tests/php/Jobs/CloneImagesOrVideosTest.php

```diff
@@ -0,0 +1,458 @@
+<?php
+
+namespace Biigle\Tests\Jobs;
+
+use Biigle\Jobs\CloneImagesOrVideos;
+use Biigle\Jobs\ProcessNewVolumeFiles;
+use Biigle\MediaType;
+use Biigle\Role;
+use Biigle\Tests\ImageAnnotationLabelTest;
+use Biigle\Tests\ImageAnnotationTest;
+use Biigle\Tests\ImageLabelTest;
+use Biigle\Tests\ImageTest;
+use Biigle\Tests\ProjectTest;
+use Biigle\Tests\VideoAnnotationLabelTest;
+use Biigle\Tests\VideoAnnotationTest;
+use Biigle\Tests\VideoLabelTest;
+use Biigle\Tests\VideoTest;
+use Biigle\Tests\VolumeTest;
+use Carbon\Carbon;
+use Illuminate\Http\Request;
+use Illuminate\Http\UploadedFile;
+use Illuminate\Support\Facades\Storage;
+use Queue;
+
+class CloneImagesOrVideosTest extends \ApiTestCase
+{
+    public function testCloneVolume()
+    {
+        $volume = $this->volume(
+            ['created_at' => '2022-11-09 14:37:00',
+                'updated_at' => '2022-11-09 14:37:00',])->fresh(); // Use fresh() to load even the null fields.
+        // The target project.
+        $project = ProjectTest::create();
+
+        $this->beAdmin();
+        $project->addUserId($this->admin()->id, Role::adminId());
```

Comment:
You don't need this any more. This is only required for the authorization checks when the API requests are tested (in `VolumeControllerTest`).